### ISSUE TITLE
"Use headings and spacing" refers to unsuitable SCs

### ISSUE THREAD

**Issue by JAWS-test** [w3c/wai-website#726]
[Use headings and spacing](https://www.w3.org/WAI/tips/designing/#use-headings-and-spacing-to-group-related-content) makes sense for accessibility, but there is no WCAG criterion that matches the description, so the reference to SC 2.4.6 and 2.4.10 is misleading as they require meaningful headings, but no layout with white spaces, etc. 
For instance, "Little spacing and unclear relationship" would not violate 2.4.6 and 2.4.10.
Therefore I suggest to remove these two links in "More information" or to insert a note indicating that 2.4.6 and 2.4.10 require something else.
